### PR TITLE
[Kyrian] - Understanding Celo's Approach to Layer-2 Scaling

### PR DESCRIPTION
**Pull request by Kyrian212**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: blog/2023-03-14-understanding-celos-approach-to-layer2-scaling/understanding-celos-approach-to-layer2-scaling.md [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+## Understanding Celo's Approach to Layer-2 Scaling: An Evaluation of Rollups, Sidechains, and State Channels
+
+
+As the demand for blockchain technology continues to grow, the issue of scalability has become increasingly important. Celo, a mobile-first blockchain platform, has implemented a variety of layer-2 scaling solutions to address this challenge, including rollups, sidechains, and state channels.
+
+
+This article provides a comprehensive evaluation of each of these approaches, explaining how they work. The article also explores the implications of Celo's layer-2 scaling solutions for the wider blockchain ecosystem, including how they may impact the user experience and the development of decentralized applications. The article is intended for developers, blockchain enthusiasts, and anyone interested in understanding how Celo's layer-2 scaling solutions are driving innovation in the blockchain industry.
+
+
+Here's an overview of what we'll cover
```
Comment (Julius170) [inline blog/2023-03-14-understanding-celos-approach-to-layer2-scaling/understanding-celos-approach-to-layer2-scaling.md:26]: add a ":" after this statement

File: blog/2023-03-14-understanding-celos-approach-to-layer2-scaling/understanding-celos-approach-to-layer2-scaling.md [hunk 2]
```diff
@@ -0,0 +1,378 @@
...
+
+
+State channels allow for off-chain transactions between parties, reducing transaction fees and increasing transaction speeds. In order to implement state channels on the Celo network, we will use the Solidity library for state channels called `celo-monorepo`.
+
+
+Here's an example of how to implement state channels on the Celo network using the celo-monorepo library:
+
+
+```
+solidity
```
Comment (Julius170) [inline blog/2023-03-14-understanding-celos-approach-to-layer2-scaling/understanding-celos-approach-to-layer2-scaling.md:254]: Solidity should come right after the 3 backslashes like this
"```solidity"

File: blog/2023-03-14-understanding-celos-approach-to-layer2-scaling/understanding-celos-approach-to-layer2-scaling.md [hunk 3]
```diff
@@ -0,0 +1,378 @@
...
+
+
+## Conclusion
+
+
+That brings us to the end of today's topic on “Understanding Celo’s approach to Layer-2 scaling”.
+To recap, you can find a summary of all the topics we covered below.
+
+
+✅ Introduction to Celo’s approach to Layer-2 scaling
```
Comment (Julius170) [inline blog/2023-03-14-understanding-celos-approach-to-layer2-scaling/understanding-celos-approach-to-layer2-scaling.md:345]: This list is quite irrelevant , There might be a feature on the celo academy that gives a summary of the headers to help it readers navigate the tutorial better
But either way You should remove this list from conclusion and add a more descriptive summary of the whole tutorial

**Review by Julius170 — COMMENTED** [review]
Hi @Kyrian212 You can now make changes based on the corrections I've spotted out

**Comment by Kyrian212** [comment]
Resolved

**Review by developerkunal — APPROVED** [review]
Approved @Ishanpathak1

**Pull request merged**